tla/consistency/SingleNode.tla

```diff
@@ -95,16 +95,24 @@ RwTxResponseAction ==
 \* Note that a request could only be committed if it's in the highest view's ledger
 StatusCommittedResponseAction ==
     /\ \E i \in DOMAIN history :
-        /\ history[i].type = RwTxResponse
-        /\ Len(ledgerBranches[Len(ledgerBranches)]) >= history[i].tx_id[2]
-        /\ ledgerBranches[Len(ledgerBranches)][history[i].tx_id[2]].view = history[i].tx_id[1]
-        \* Reply
-        /\ history' = Append(
-            history,[
+        LET view == history[i].tx_id[1]
+            seqno == history[i].tx_id[2]
+        IN /\ history[i].type = RwTxResponse
+           /\ Len(Last(ledgerBranches)) >= seqno
+           /\ Last(ledgerBranches)[seqno].view = view
+           \* There is no future InvalidStatus that's incompatible with this commit
```

Comment:
I don't understand this comment because it says "future" but quantifies over the (unprimed) *history*.

Reply:
That's future in the ledger sense, i.e. a tx on the same view but with a higher (later) seqno. Perhaps using the word later is preferable here.